odbc_connection.c

```diff
@@ -817,6 +818,17 @@ odbc_connect_new (ODBC_CONNECTION * conn,
   pt = charset == NULL ? "" : charset;
   conn->charset = UT_MAKE_STRING (pt, -1);
 
+  ut_tolower (autocommit);
+  pt = autocommit == NULL ? "on" : autocommit;
+  if (strcmp (autocommit, "on") == 0)
```

Comment:
Suggestion: remove function **ut_tolower ()** and use case independent string compare,  **_strcmp(),** as other codes do, if it is possible.

```
  pt = autocommit == NULL ? "on" : autocommit;
  if (_strcmp (autocommit, "on") == 0)
```

Reply:
ut_tolower() is to compare the strings input in AUTOCOMMIT without case-sensitive.
Are you saying it should be case sensitive?

Reply:
**_strcmp()** is case insensitive string compare.

Reply:
My mistake it was
**_stricmp ()**

Reply:
```
  pt = autocommit == NULL ? "on" : autocommit;
  if (_stricmp (autocommit, "on") == 0)
```